.github/workflows/lint.yml

```diff
@@ -29,7 +29,3 @@ jobs:
         with:
           version: 3.44.1
           repo-token: ${{ secrets.GITHUB_TOKEN }}
-
```

Comment:
the CI won't run `task lint`, so this would be lost. is that what you want?

Reply:
Good catch. I'll fix the workflow to run `task lint`